### PR TITLE
HOTT-1537: Tiered admin rights

### PR DESCRIPTION
**Pull request by maverickvi**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-1537

### What?

I have added/removed/altered:

- [x] Added "HMRC Admin" role to admin 

### Why?

I am doing this because:

- We don't want to give full admin access to GDS and HMRC editors

Screenshot 2023-02-22 at 14 32 42
Screenshot 2023-02-22 at 14 32 31
Screenshot 2023-02-22 at 14 32 15
Screenshot 2023-02-22 at 14 31 31
Screenshot 2023-02-22 at 14 31 01

### CODE REVIEW

**Review by willfish — APPROVED** [review]

**Comment by willfish** [comment]
Looks like you could do with updating the signon user that's associated with the smoke tests to full access

**Review by jebw — APPROVED** [review]
LGTM

Comment in PR description can now be updated

File: app/policies/tariff_update_policy.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+TariffUpdatePolicy = Struct.new(:user, :tarif_update) do
```
Comment (willfish) [inline app/policies/tariff_update_policy.rb:1]:
```suggestion
TariffUpdatePolicy = Struct.new(:user, :tariff_update) do
```

**Review by jebw — APPROVED** [review]
LGTM

**Pull request merged**